tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -56,15 +56,15 @@ static void to_display_format(const u8_t *src, size_t size, char *dst)
 
 struct spi_config spi_cfg_slow = {
 	.frequency = SLOW_FREQ,
-	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
+	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL | SPI_MODE_LOOP |
```

Comment:
Actually no. SPI_MODE_LOOP is different than what this test is meant for. This test is meant to run on hardware with a wire short-cutting MISO-MOSI lines. So it will act as if there is an actual hw slave echoing everything being sent.
SPI_MODE_LOOP mode works slightly differently depending on the controller.

That said, it would be indeed nice to test SPI_MODE_LOOP if requested by the user.
Could you modify this commit so SPI_MODE_LOOP would be conditioned upon a local Kconfig option?
(see tests/drivers/spi/spi_loopback/Kconfig)

Reply:
Thanks @tbursztyka. This makes sense.